CHANGELOG.rst

```diff
@@ -13,6 +13,8 @@ Change Log
 - Add migration for django_comments
 - Upgrade Django to 1.8.14
 - Upgrade to django-tinymce 2.4.0 (Mr. Senko)
+- Refactor tcms.core.migrations.0001_django_comments__object_pk.
+  PR #157 (Mr. Senko)
```

Comment:
Part ``#157`` is not in the summary.

When make new release, I'll make a full list of change history. I think it is not necessary to change this file every time along with a patch.